src/core_write.cpp

```diff
@@ -220,8 +220,27 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
             in.pushKV("txinwitness", txinwitness);
         }
         if (calculate_fee) {
-            const CTxOut& prev_txout = txundo->vprevout[i].out;
+            const Coin&   prev_coin  = txundo->vprevout[i];
+            const CTxOut& prev_txout = prev_coin.out;
+
             amt_total_in += prev_txout.nValue;
+            switch (verbosity) {
```

Comment:
personally think it's clearer to comprehend, compared to something like `verbosity == TxVerbosity::SHOW_TXID || verbosity == TxVerbosity::SHOW_DETAILS)`

Reply:
But here it would be `if (verbosity == TxVerbosity::SHOW_DETAILS_AND_PREVOUT) { /* compute something */ }`. Seems very straightforward and Bitcoin Core does not use switches instead of simple if statements by default.

Obviously, this is rather a nit. So I don't want to waste your time with this. Feel free to resolve this suggestion if you are happy with the code as it is.